src/menu/configMenu.c

```diff
@@ -88,24 +88,34 @@ void configMenu()
 			setAutoResume(!autoResumeEnabled());
 			redraw = true;
 		}
-/*		if(vpad.trigger & VPAD_BUTTON_LEFT)
+		if(vpad.trigger & VPAD_BUTTON_LEFT)
 		{
-			int l = getUnfilteredLanguage();
-			if(--l < Swkbd_LanguageType__Japanese)
-				l = Swkbd_LanguageType__Invalid;
-			
-			setKeyboardLanguage(l);
+			char* l = getFormattedRegion(getRegion());
+			if(l == "All") {

```

Comment:
What you do here (`==` with strings) is undefined behaviour. You could compare the `#define`d strings with `==` instead through, like this: `if(l == SET_ALL)`.

Reply:
I think I did this for better code readability and for displaying the region on the menu (?), I’m not totally sure tho